projects/spectator/src/lib/spectator-with-routing/router-link-stub.ts

```diff
@@ -0,0 +1,26 @@
+import { forwardRef, Directive, HostListener, Input } from '@angular/core';
+import { RouterLink } from '@angular/router';
+
+// tslint:disable
+
+@Directive({
+  selector: '[routerLink]',
+  providers: [
+    {
+      provide: RouterLink,
+      useExisting: forwardRef(() => RouterLinkDirectiveStub)
```

Comment:
When you do it inline, you don't need `forwardRef`.

Reply:
👍